Makefile

```diff
@@ -0,0 +1,50 @@
+# text reset
+NO_COLOR=\033[0m
+# green
+OK_COLOR=\033[32;01m
+# red
+ERROR_COLOR=\033[31;01m
+# cyan
+WARN_COLOR=\033[36;01m
+# yellow
+ATTN_COLOR=\033[33;01m
+
+ROOT_DIR := $(shell git rev-parse --show-toplevel)
+
+VERSION := `git describe --tags --dirty 2>/dev/null`
+COMMITHASH := `git rev-parse --short HEAD 2>/dev/null`
+DATE := `date "+%FT%T%z"`
+
+.PHONY: all
+all: init test
+
+init:
+	@echo "$(ATTN_COLOR)==> init $(NO_COLOR)"
+
+.PHONY: test
+test:
+	@echo "$(ATTN_COLOR)==> test $(NO_COLOR)"
+	@echo "Running Unit test"
+	@mvn package -P Unittest

```

Comment:
These should be mvn verify rather than package

Reply:
Duly noted, it will reflect in the final commit along with the before mentioned change.